libs/src/api/playlist.js

```diff
@@ -137,7 +138,38 @@ class Playlists extends Base {
       }
     }
 
-    return this.contracts.PlaylistFactoryClient.orderPlaylistTracks(playlistId, trackIds)
+    return this.contracts.PlaylistFactoryClient.orderPlaylistTracks(playlistId, trackIds, retries)
+  }
+
+  /**
+   * Checks if a playlist has entered a corrupt state
```

Comment:
nit: could we write a few words about in what sense the playlist is corrupt? this is really checking for a very specific degenerate case + I will for sure have forgotten what that is when I look at this 6 mo from now :)

Reply:
gotcha